js/data.js

```diff
@@ -49,35 +49,46 @@ const _FACTIONS = {
 
 const _MAPS = {
   WOODLAND: {
-    name: "Woodland",
+    name: "Fall",
+    alt: "The Fall Root map.",
+    fileName: "fall.png",
```

Comment:
fileName is kind of ambiguous, maybe rename to "image" or "imageFileName" or something?

Reply:
Ok

Reply:
done